cnd/src/ethereum.rs

```diff
@@ -1,22 +1,140 @@
 #![warn(unused_extern_crates, missing_debug_implementations, rust_2018_idioms)]
 #![forbid(unsafe_code)]
 
+use bitcoin::hashes::core::{fmt::Formatter, str::FromStr};
```

Comment:
Why are we importing these via `bitcoin`?  Can they not come from `std`?

Reply:
oh, good catch, fixing :)